src/include/migraphx/generate.hpp

```diff
@@ -33,38 +33,48 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
+enum class MIGRAPHX_EXPORT normalize_range
```

Comment:
I would suggest to call this `random_mode` and then call it `legacy`(for the current behaviour) and `uniform`

Reply:
There could be other schemes that would also be `uniform`. One could argue, the `legacy` -- aka 32 number only in the 32-bit universe -- are also uniformly spread ;-) 

But if you still like the names `legacy` and `uniform` very much, @pfultz2, let me know, this is not a big issue for me.

Reply:
> There could be other schemes that would also be uniform.

Sure and if we want to switch between them we can name them uniquely such as `uniform`(for the default) and `uniform_zebra`(and for when we want to use zebras).

>  One could argue, the legacy -- aka 32 number only in the 32-bit universe 

32 numbers is 5 bit, and that only applies to floats as integers uses a different scheme. 

> -- are also uniformly spread ;-)

Kind of like a quantized uniform, but either way I would like to remove this at some point, which is why I was thinking `legacy`. We could possibly do `uniform_legacy` maybe if you think thats a better name.